parmed/__init__.py

```diff
@@ -6,7 +6,7 @@
 # Version format should be "major.minor.patch". For beta releases, attach
 # "-beta#" to the end. The beta number will be turned into another number in the
 # version tuple
-__version__ = '2.0.12'
+__version__ = '2.1.0'
```

Comment:
The `parmed.topologobjects.Group` API changed a little.  Ergo, bigger version bump.